### PR TITLE
Visual tweaks for reports, graphs and UI components

### PR DESCRIPTION
**Pull request by moreofmorris**
### Purpose
This is linked with this [JIRA ticket](https://buffer.atlassian.net/browse/ANA-124). This will be a rather large pull request which includes many small (but necessary!) styling tweaks to help improve the appearance of reports and the app.

Some highlights include:
- Tighter spacing and layouts for charts and pages
- More considered font sizes and weights across the app
- Redesigned post breakdown table
- Redesigned reports page
- More consistent buttons and dropdown
- A zillion other little things

### Notes
See the latest here:
https://task-improved-reports-design-analyze.dev.buffer.com/

None of my changes should be changing logic anywhere, it's all visual stuff but does involve converting some components over to styled components for example. I do know I have some hardcoded colours in here but I will address a lot of this with more planned pull requests, hope that's ok.

I'm super sorry this is such a big pull request but one thing led to another. The task ballooned into many different directions when I realised so much needed to be fixed. It would be nice that the refresh hits the customer in one go though? 🤔

### CODE REVIEW

**Comment by tiggreen** [comment]
Hey @moreofmorris before I jump on the code review I think something is off with the post-breakdown table rendering. http://hi.buffer.com/143Y2t2Q0I2E. it does render slow and when I export to PDF it renders it as blank. I thought I'd tell you or I can try to find out why when I review the code.

File: packages/profile-selector/components/ProfileSelectorDropdown/index.jsx [hunk 1]
```diff
@@ -1,27 +1,35 @@
...
 } from '@bufferapp/components';
 
-const dropdownContainerStyle = {
-  position: 'relative',
-  display: 'inline-block',
-  zIndex: 1,
-  width: '360px',
-  height: '34px',
-};
+const DropdownContainer = styled(Dropdown)`
```
[the remark this answers is not in the document]
Comment (moreofmorris) [reply packages/profile-selector/components/ProfileSelectorDropdown/index.jsx]: It's so awesome, right? I only found this out very recently. So useful too!

File: packages/compare-chart/components/CompareChart/index.jsx [hunk 2]
```diff
@@ -28,6 +28,17 @@ const Container = styled.div`
   width: 100%;
 `;
 
+const Header = styled.div`
```
[the remark this answers is not in the document]
Comment (moreofmorris) [reply packages/compare-chart/components/CompareChart/index.jsx]: I plan to do a lot more! I kind of weirdly enjoy the process! :)

File: packages/add-report/package.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
   "author": "mike@msanroman.io",
   "dependencies": {
     "@bufferapp/analyze-shared-components": "^0.60.12",
-    "@bufferapp/components": "2.1.1",
+    "@bufferapp/components": "2.1.1-alpha.6",
```
[the remark this answers is not in the document]
Comment (moreofmorris) [reply packages/add-report/package.json]: Ah, yep. I need to publish a new version there. Perfect, will do that today 👍

**Pull request merged**